sdk/websites/Microsoft.Azure.Management.WebSites/src/Properties/AssemblyInfo.cs

```diff
@@ -8,8 +8,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Web Sites Management Library")]
 [assembly: AssemblyDescription("Provides management functionality for Microsoft Azure Web Sites.")]
 
-[assembly: AssemblyVersion("3.1.1")]
-[assembly: AssemblyFileVersion("3.1.1")]
+[assembly: AssemblyVersion("3.1.2")]
```

Comment:
since this is major API version change should bump major version? Hi @allenjzhang what's the normal practice? thanks

Reply:
> since this is major API version change should bump major version? Hi @allenjzhang what's the normal practice? thanks

@yungezz / @allenjzhang ping any update on this - can this be merged or is there a need to update the Api Version?

Reply:
@Kotasudhakarreddy  @panchagnula If it's not a fix release, I suggest to bump the version to `3.2.0`. See the [semantic versioning](https://semver.org/)

Reply:
We can bump the version up to 3.2.0, due to the API verison change. Other than that, this PR looks good to me.